### ISSUE TITLE
Values in LineSet not displayed correctly if LINE_MIN > 0

### ISSUE THREAD

**Issue by TakilianRueshin** [diogobernardino/WilliamChart#23]
```java
private final static String[] lineLabels = {"JAN", "FEB", "MAR", "APR", "MAY", "JUN", "JUL"};
 private final static float[][] lineValues = { {201f, 202f, 210f, 220f, 222.5f, 222.5f, 230f},
            {-9f, -2f, -4f, -3f, -7f, -5f, -3f}};
    private final static int LINE_MAX = 240;
    private final static int LINE_MIN = 200;
    private final static int[] beginOrder = {0, 1, 2, 3, 4, 5, 6};

    @AfterViews
    public void setup(){
        Paint mLineGridPaint = new Paint();
        mLineGridPaint.setColor(this.getResources().getColor(R.color.primary_light));
        mLineGridPaint.setPathEffect(new DashPathEffect(new float[] {5,5}, 0));
        mLineGridPaint.setStyle(Paint.Style.STROKE);
        mLineGridPaint.setAntiAlias(true);
        mLineGridPaint.setStrokeWidth(Tools.fromDpToPx(.75f));

        mainChart.reset();
        mainChart.setOnEntryClickListener(lineEntryListener);
        LineSet dataSet = new LineSet();
        dataSet.addPoints(lineLabels, lineValues[0]);
        dataSet.setDots(true)
                .setDotsColor(this.getResources().getColor(R.color.light_green_200))
                .setDotsRadius(Tools.fromDpToPx(5))
                .setDotsStrokeThickness(Tools.fromDpToPx(2))
                .setDotsStrokeColor(this.getResources().getColor(R.color.light_green_400))
                .setLineColor(this.getResources().getColor(R.color.light_green_400))
                .setLineThickness(Tools.fromDpToPx(3));
        mainChart.addData(dataSet);

        mainChart.setBorderSpacing(Tools.fromDpToPx(4))
                .setHorizontalGrid(mLineGridPaint)
                .setXAxis(true)
                .setXLabels(XController.LabelPosition.OUTSIDE)
                .setYAxis(true)
                .setYLabels(YController.LabelPosition.OUTSIDE)
                .setAxisBorderValues(LINE_MIN, LINE_MAX, 10)
                .setLabelsMetric("kg")
                .show()
        //.show()
        ;
    }
```

```xml
   <com.db.chart.view.LineChartView

        xmlns:chart="http://schemas.android.com/apk/res-auto"
        android:id="@+id/mainChart"
        android:layout_below="@+id/sectionPrograms"
        android:layout_centerInParent="true"
        android:layout_width="match_parent"
        android:layout_height="200dp"
        android:layout_marginTop="7dp"
        android:layout_marginLeft="7dp"
        android:layout_marginRight="7dp"
        android:layout_marginBottom="2dp"
        android:paddingTop="10dp"
        android:paddingBottom="10dp"
        android:paddingLeft="10dp"
        android:paddingRight="10dp"
        android:background="#00FFFFFF"
        chart:chart_axisColor="@color/primary"
        chart:chart_labelColor="@color/secondary_text"
        chart:chart_axisBorderSpacing="0dp"></com.db.chart.view.LineChartView>
```

Result in this:
![5554 lollipop_nexus_5_018](https://cloud.githubusercontent.com/assets/402236/5577712/d5023e98-9017-11e4-8eb0-9c69853fbf3c.png)

If I set LINE_MIN = 0 all values displayed correctly

What I'm doing wrong?

Thanks for the help!

**Comment by diogobernardino** [comment]
Hi. Can't see any problem looking at your code, I will need to test it myself and try to reproduce the problem.

**Comment by xiongwei-git** [comment]
I share the same issue with TakilianRueshin

**Comment by diogobernardino** [comment]
Fix #25 by @TakilianRueshin merged. Many thanks.
